packages/ag-charts-community/src/chart/series/polar/pieSeries.ts

```diff
@@ -379,7 +379,7 @@ export class PieSeries extends PolarSeries<PieNodeDatum, Sector> {
                     max: this.radiusMax,
                 }),
                 valueProperty(this, radiusKey, true, { id: `radiusRaw` }), // Raw value pass-through.
-                normalisePropertyTo(this, { id: 'radiusValue' }, [0, 1], this.radiusMin ?? 0, this.radiusMax)
+                normalisePropertyTo(this, { id: 'radiusValue' }, [0, 1], 1, this.radiusMin ?? 0, this.radiusMax)
```

Comment:
It's a shame javascript doesn't have pythons optionally named params, like `normalisePropertyTo(this, { ... }, zeroDomain = 1)`

Reply:
Agreed :-(